stdlib/log/logger.lua

```diff
@@ -76,6 +79,9 @@ function Logger.new(mod_name, log_name, debug_mode, options)
                 return Logger.write()
             end
         else
+            if Logger.options.use_log then
+                log(format("%s/%s: %s", Logger.mod_name, Logger.log_name, msg))
+            end
             table.insert(Logger.buffer, format("00:00:00: %s\n", msg))
```

Comment:
Not sure how it would be leaking memory. In the data stage it should be freed once the lua environment get's destroyed, in control.lua it would be written eventually? Anyway the latest commit changes it.